doc/game.md

```diff
@@ -5,20 +5,26 @@ The Game Loop module is a simple abstraction over the game loop concept. Basical
 * The render method takes the canvas ready for drawing the current state of the game.
 * The update method receives the delta time in seconds since last update and allows you to move to the next state.
 
-The class `Game` can be subclassed and will provide both these methods for you to implement. In return it will provide you with a `widget` property that returns the game widget, that can be rendered in your app.
+The class `Game` can be subclassed and will provide among others, these gameloop methods.
+At the same time, a `Game` subclass instance can be added to a Flutter widget tree via the `GameWidget`.
 
-You can either render it directly in your `runApp`, or you can have a bigger structure, with routing, other screens and menus for your game.
+You can add it into the topo of you tree (directly as an argument to `runApp`) or inside the usual app-like widget structure widget structure (with scaffold, routes and whatnot).
 
-To start, just add your game widget directly to your runApp, like this:
+Example of usage directly into `runApp`:
 
 ```dart
     main() {
-        Game game = MyGameImpl();
-        runApp(game.widget);
+        runApp(
+          GameWidget(
+            game: MyGameSubClass()
+          )
+        );
     }
 ```
 
-Instead of implementing the low level `Game` class, you should probably use the more full-featured `BaseGame` class, or the `Forge2DGame` class if you want to use a physics engine.
+It is important to notice that `Game` is an abstract class with just the very basic implementations of the gameloop.
+
+As an option and more suitable for most cases, there is the full-featured `BaseGame` class. For advanced the Forge2D physics engine use `Forge2DGame` class;
```

Comment:
This "for advanced the Forge2d" is sounding strange, not sure what you meant here